model-optimizer/extensions/front/tf/while_ext.py

```diff
@@ -58,7 +58,7 @@ def update_body_graph(body_graph: Graph, subgraph_proto: dict,
                 'out': src_port,
                 'in': dst_port,
                 'name': src_id,
-                'fw_tensor_debug_info': [(src_id, src_port)],
+                'fw_tensor_debug_info': [(src_id, dst_port, inp)],
```

Comment:
please pay attention that inp may be equal to `functional_1/my_while_loop/while/add_1/y:output:0`, for example.  So I am not sure that you want exactly this one. I propose to consider `orig_src_id` instead.

Reply:
I expect that inp is tensor name from framework and it should contain ":<port idx>" in the end. orig_src_id does not contain port idx, so it is not what needed here.

Reply:
I see your point. But pay attention to that `inp` can contain `output` between two columns. And you may need `orig_src_id + ':' + src_port` then.

Reply:
Yes, if there is output, then it is not what I need. Changed to orig_src_id + ':' + src_port.